### PR TITLE
Add react router

### PR DESCRIPTION
**Pull request by wsakolski**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -15,24 +15,11 @@
 
 .App-header {
   background-color: #282c34;
```
Comment (sylwiaszymanska) [inline src/App.css]: Add color to variables file :)

File: src/App.css [hunk 2]
```diff
@@ -15,24 +15,11 @@
...
-  min-height: 100vh;
+  min-height: 20vh;
```
Comment (sylwiaszymanska) [inline src/App.css]: Add variable to theme

Comment (wsakolski) [reply src/App.css]: thanks 🥇

File: src/App.css (hunk 1 above)
Comment (wsakolski) [reply src/App.css]: Ok, maybe next week :v

**Pull request merged**